scripts/weather_wrapper.sh

```diff
@@ -20,7 +20,7 @@ main()
 
   if [ "$(expr ${TIME_LAST} + ${RUN_EACH})" -lt "${TIME_NOW}" ]; then
     # Run weather script here
-    $current_dir/weather.sh $fahrenheit $location $fixedlocation > "${DATAFILE}"
+    $current_dir/weather.sh $fahrenheit $location "$fixedlocation" > "${DATAFILE}"
```

Comment:
Couldn't this just be `"${fixed_location}"`? https://stackoverflow.com/questions/8748831/when-do-we-need-curly-braces-around-shell-variables

Reply:
(duplicate comment maybe ?)

yes, it's the same with my changes